components/brave_webtorrent/extension/components/mediaViewer.tsx

```diff
@@ -7,9 +7,47 @@ import * as React from 'react'
 // Constants
 import { TorrentObj, File } from '../constants/webtorrentState'
 
-const SUPPORTED_VIDEO_EXTENSIONS = ['m4v', 'mkv', 'mov', 'mp4', 'ogv', 'webm']
-
-const SUPPORTED_AUDIO_EXTENSIONS = ['aac', 'mp3', 'ogg', 'wav', 'm4a']
+const SUPPORTED_VIDEO_EXTENSIONS = [
+  'm4v',
+  'mkv',
+  'mov',
+  'mp4',
+  'ogv',
+  'webm'
+]
+
+const SUPPORTED_AUDIO_EXTENSIONS = [
+  'aac',
+  'flac',
+  'm4a',
+  'm4b',
+  'm4p',
+  'mp3',
+  'oga',
+  'ogg',
+  'wav'
+]
+
+const SUPPORTED_IMAGE_EXTENSIONS = [
+  'bmp',
+  'gif',
+  'jpeg',
+  'jpg',
+  'png',
+  'svg'
+]
+
+const SUPPORTED_PDF_EXTENSIONS = [
+  'pdf'
+]
```

Comment:
PDFs are no longer displayed in an iframe, so we break them into their own type.